templates/rustdoc/topbar.html

```diff
@@ -134,11 +134,8 @@
                             <li class="pure-menu-heading">Versions</li>
 
                             <li class="pure-menu-item">
-                                <div class="pure-menu pure-menu-scrollable sub-menu" tabindex="-1">
-                                    <ul class="pure-menu-list">
-                                        {# Display all releases of this crate #}
-                                        {{ macros::releases_list(name=krate.name, releases=krate.releases, target=target, inner_path=inner_path) }}
-                                    </ul>
+                                <div class="pure-menu pure-menu-scrollable sub-menu" id="releases-list" tabindex="-1">
+                                    <span class="rotate">↻</span>
```

Comment:
I'll use this one then: https://fontawesome.com/v6/icons/spinner?f=classic&s=solid

Reply:
It could have a little bit of left margin, but it spins centered now

<img width="281" alt="image" src="https://user-images.githubusercontent.com/81079/236003611-c1e96c44-e763-435f-852a-90a6e83794b2.png">